scripts/build.py

```diff
@@ -201,6 +212,12 @@ def _minify(source_path, target_path):
     # experiments, 18m seems to work, but 12m is too small and results in an
     # out-of-memory error.
     # https://circleci.com/blog/how-to-handle-java-oom-errors/
+    # Use relative path to avoid java command line parameter parse error on
+    # Windows.
+    target_path = common.convert_filepath_to_url(
```

Comment:
The variables need to be renamed, too. What do they represent exactly? (In general don't change the meaning of a variable midstream -- here, target_path is being redefined.)

Looking at the usage, I'm not entirely sure that the result is a URL either, since presumably it doesn't include a protocol. Could you maybe explain what exactly the arguments after -o represent?

Reply:
Oh, I was in a rush and this place is why I called the function `convert_windows_style_path_to_unix_style` originally.

The redefined `target_path` is kind like a normalized path - it should be normalized to Unix-style to avoid java's syntax error. Same for the redefined `source_path`.

Reply:
OK -- given that, I think you can use `convert_to_posixpath(filepath)` instead.